src/schema/graph_schema.json

```diff
@@ -2,19 +2,40 @@
   "node": {
     "type": "object",
     "properties": {
-      "degree": {"type": "number"},
-      "x": {"type": "number"},
-      "y": {"type": "number"},
-      "version": {"type": "string"},
-      "image": {"type": "string"},
-      "bitcoin_config": {"type": "string", "default": ""},
-      "tc_netem": {"type": "string"},
-      "exporter": {"type": "boolean", "default": false},
-      "collect_logs": {"type": "boolean", "default": false},
-      "build_args": {"type": "string", "default": ""},
-      "ln": {"type": "string"},
-      "ln-image": {"type": "string"},
-      "ln-cb-image": {"type": "string"}
+      "version": {
+        "type": "string",
+        "comment": "Bitcoin Core version with an available Warnet tank image on Dockerhub. May also be a GitHub repository with format user/repository:branch to build from source code"},
```

Comment:
Really like these comments! Didn't know that was an option but makes sense that it should be one.